app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -7,9 +7,16 @@
 
 .govuk-govspeak {
   .example {
-    border-left: 10px solid $panel-colour;
-    padding-left: 1.5em;
-    margin: 2.5em 0;
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
```

Comment:
> I'd prefer we redefine our spacing system to use em rather than be inconsistent.
That's a big change.

For now, using ems in this case improves legibility and makes spacing consistent with other content.

We're using the grid classes to define vertical margins on paragraphs etc though:
https://github.com/alphagov/static/blob/master/app/assets/stylesheets/govuk-component/govspeak/_typography.scss#L16-L17